index.js

```diff
@@ -60,23 +20,43 @@ function stack(results) {
 		i++;
 
 		log.writelpad(chalk.red(i + '.', result.title));
-		log.writelpad(chalk.red(beautifyStack(result.error.stack)));
+		log.stack(result.error.stack);
 		log.write();
 	});
 }
 
+function test(err, title, duration) {
+	if (isForked) {
+		// serialize Error object
+		if (err) {
+			err = {
+				message: err.message,
+				stack: err.stack
+			};
+		}
+
+		process.send({
+			err: err || {},
+			title: title,
+			duration: duration
+		});
+
+		return;
+	}
+
+	log.test(err, title, duration);
```

Comment:
Hmm, I think we're way past supporting `node test.js` now, with parallelism and ES2015. Unless you can come up with a good way to support both cases. Maybe detect that it's executed directly and log output. You can detect direct execution with: `require.main === module`.